pkg-builder.pl

```diff
@@ -456,7 +456,10 @@ ()
    cpy_file( "store/build/dist/zm-store.jar",                    "$stage_base_dir/opt/zimbra/lib/jars/zimbrastore.jar");
    cpy_file( "soap/build/dist/zm-soap.jar",                      "$stage_base_dir/opt/zimbra/lib/jars/zimbrasoap.jar");
    cpy_file( "client/build/dist/zm-client.jar",                  "$stage_base_dir/opt/zimbra/lib/jars/zimbraclient.jar");
-
+   cpy_file( "client/build/dist/zm-comnon.jar",                  "$stage_base_dir/opt/zimbra/lib/jars/zimbracommon.jar");
+   cpy_file( "client/build/dist/zm-native.jar",                  "$stage_base_dir/opt/zimbra/lib/jars/zimbranative.jar");
+   cpy_file( "client/build/dist/zm-common.jar",                  "$stage_base_dir/opt/zimbra/lib/jars/zimbracommon.jar");
```

Comment:
looks duplicate. 
line 459 and 461. 
line 460 and 462. 

zimbranative.jar or zimbra-native.jar : just a double check.

Reply:
Actually 459 has a typo in it.

Reply:
Fixed